packages/components/src/components/contract-card/result-overlay/result-overlay.tsx

```diff
@@ -45,20 +43,11 @@ const ResultOverlay = ({
     getCardLabels,
     getContractPath,
     is_positions,
-    is_unsupported,
     is_visible,
-    onClick,
     onClickRemove,
     payout_info,
     result,
 }: TResultOverlayProps) => {
-    const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
Unsupported contracts and forward starting, which haven't started yet, won't be shown in the Recent Positions (because I added filtration), so no need in all this logic